yarn.lock

```diff
@@ -4924,10 +4924,10 @@ wbuf@^1.1.0, wbuf@^1.7.3:
   dependencies:
     minimalistic-assert "^1.0.0"
 
-webcomponents.js-2@^0.7.24:
+webcomponents.js@^0.7.24:
```

Comment:
Another open pull request already has this change so if you have time it would be great to rebase without this file to prevent possible merge issues.

Reply:
Will do. Must have been updated when I was building things.